### DISCUSSION TITLE
How do I get Deadly Goo back?

### DISCUSSION THREAD

**Discussion by kingkyroLK** [BEEmod/BEE2-items#4188] [Tech Support / Q&A] [answered]
Whenever I place it and build the Test Chamber it always appears as a Bottomless pit. I don't hate the Bottomless pit, but I want to use the Deadly goo every once in awhile, I know there's probably an option to swap them, but I can't find it. Can anyone help?

**Comment by saismeesaimonsaimes** [comment, marked as answer]
Change the skybox in the application. Any 3D skyboxes will automatically generate bottomless pits.

**Comment by vrad-exe** [comment]
~First of all, please don't open issues to ask questions, this isn't a forum.~ You're also using an old version of BEE2.4, this issue tracker is only for the latest version (currently 4.39.2) and development versions.
